### PR TITLE
Adding support for user password migration.

### PR DESCRIPTION
**Pull request by joshirohit100**
This will allow migrating user password from MD5/Plain text format.

### CODE REVIEW

**Comment by joshirohit100** [comment]
cc @pranitjha @amitgoyal

File: src/Plugin/DestinationType/Users.php [hunk 1]
```diff
@@ -21,6 +21,7 @@ class Users extends DestinationTypeBase {
   public function getDestinationYml() {
     return [
       'plugin' => 'entity:user',
+      'md5_passwords' =>  TRUE,
```
Comment (amitgoyal) [inline src/Plugin/DestinationType/Users.php]: Does this mean that we are only importing plain passwords and converting those plain passwords to md5? OR source passwords are always in md5 and not plain text?

Comment (joshirohit100) [reply src/Plugin/DestinationType/Users.php]: This means that we support only source password of type md5. If sourcce password is of plain text, i have added  process plugin 'callback' to convert them to md5.

Comment (amitgoyal) [reply src/Plugin/DestinationType/Users.php]: sounds good.

**Comment by amitgoyal** [comment]
LGTM

**Pull request merged**